instagram/src/App.js

```diff
@@ -0,0 +1,47 @@
+import React, { Component } from 'react';
```

Comment:
You can actually import Fragment here alongside Component so you don't have to use React.Fragment in the rest of the file. 

```import React, { Component, Fragment } from 'react';```

Reply:
Thank you!